### ISSUE TITLE
Designers collaboration with NodeJS

### ISSUE THREAD

**Issue by maddhruv** [nodejs/admin#107]
This is in regard to the issue https://github.com/nodejs/badges/issues/7 in **Badges** and also a discussion for the requirement of a UI designer for the website-redesign initiative that if there are currently active designers with Nodejs or any company that Nodejs is collaborating with for our design purposes.
- The **badges** team is set for preparing a [Node.js Badges Illustration Style Guide](https://github.com/nodejs/badges/issues/7) issuing/preparing the v1 badges to roll out as also discussed in the [meeting](https://github.com/nodejs/badges/issues/16)
- Also, the **website-redesign** team is seeking help from a healthy design collaborator(person/company) to help us out with the basic color schemas, UI designs, and other design pieces of stuff.

If there are some designers with Nodejs or some scope of getting in collaboration with a designing company for a long term of support, please let us know.

/cc @nodejs/community-committee

**Comment by chowdhurian** [comment]
Thanks for putting out the word re: [nodejs/badges#7](https://github.com/nodejs/badges/issues/7)!

With regards to the second item, it could be helpful to link here the relevant issue from [nodejs/website-redesign](https://github.com/nodejs/badges/issues/7).

**Comment by maddhruv** [comment]
We don't have any issue for the same in website-redesign as of now.
But is a concern that shall be an issue at some point in the initiative.

**Comment by codeekage** [comment]
For the website-redesign, I'd love to champion that if permission is given. will probably need some links that the foundation wants on the web-page to make it a little bit better. Like specific enhancement other than the ones from my little brain.

**Comment by amiller-gh** [comment]
Since website redesign is un-blocked on this (thank you RunKit/Stripe!), and badges is blocked by website redesign's illustration guidelines (in progress), I'm going to close this issue in ~24 hours unless there are objections. We'll handle developing design standards and the process for new design contributions in the website-redesign repo as an deliverable of our design system work. If there are more admin-repo level topics to discuss after website-redesign has iterated on our design proposals, we can re-open under a new ticket 👍

**Comment by bnb** [comment]
Per @amiller-gh's comment, there have been no objections to closing this. Going to close for now 👍
